## firestore_transaction_test

### Screenshots
The before-and-after images shows that the `counter` is still incremented even when an exception has been thrown by the transaction.
<p float="left">
<img src="https://github.com/DominicOrga/firestore_transaction_test/blob/master/assets/images/before.webp" width="250"/>
<img src="https://github.com/DominicOrga/firestore_transaction_test/blob/master/assets/images/after.webp" width="250"/>
</p>
